locale/ja/about/releases.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: about-release-schedule.hbs
+title: リリース
+statuses:
+  maintenance: 'Maintenance LTS'
```

Comment:
I think these values (and the ones in `columns` below) are translated in other languages. Although I think "LTS" is sometimes left in

Reply:
@nschonni Cc @e-jigsaw 
Since it was translated in other languages, I will translate this part as well.